app/views/admin/edition_images/_uploaded_images.html.erb

```diff
@@ -38,3 +38,28 @@
     margin_top: 0
   } %>
 </div>
+
+<% if edition.images %>
+  <ul class="govuk-list">
+    <% edition.images.each_with_index do |image, index| %>
+      <li class="app-view-edition-images__row govuk-grid-row">
+        <div class="govuk-grid-column-one-third">
+          <% if image.url.present? %>
+            <img src="<%= image.url %>" alt="Image <%= index + 1 %>" class="app-view-edition-images__image">
+          <% end %>
+        </div>
+        <ul class="app-view-edition-images__details govuk-grid-column-two-thirds govuk-list govuk-list--bullet">
+          <li><strong>Caption: </strong><%= image.caption.blank? ? "None" : image.caption %></li>
+          <% if image.alt_text %><li><strong>Alt text: </strong><%= image.alt_text.blank? ? "None" : image.alt_text %></li><% end %>
+          <li><strong>Markdown code: </strong><span>[Image:&nbsp;<%=image.image_data.carrierwave_image %>]</span></li>
+        </ul>
+        <div class="app-view-edition-images__actions govuk-grid-column-full">
+          <%= link_to("Delete image", confirm_destroy_admin_edition_image_path(@edition, image), class: "govuk-link gem-link--destructive") %>
+        </div>
+      </li>
+      <% if image != edition.images.last %>
+        <li aria-hidden="true"><hr class="app-view-edition-images__section-break govuk-section-break govuk-section-break--m govuk-section-break--visible"></li>
+      <% end %>
```

Comment:
It seems like this `<hr>` exists purely for styling, rather than having a semantic meaning. I'm assuming this based on the use of `aria-hidden` attribute to hide it from accessibility tools.

In which case, is there a reason why you didn't consider implementing this visual separator in CSS?

For example, something like this:

```css
.app-view-edition-images__row {
  border-bottom: 1px solid /* some colour */;
}

.app-view-edition-images__row:last-child {
  border-bottom: 0;
}
```

I have a feeling you might have already tried this, given the existence of the `app-view-edition-images__row` class name on the main `<li>` elements. It'd certainly be nice to avoid the HTML bloat, if possible. Did you already try it, and was there a reason why you landed on the additional `<li>` and `<hr>` instead?

If the reasoning is maintain consistency with existing Design System styles, it might be good enough to simply [re-use the same colour variable](https://github.com/alphagov/govuk-frontend/blob/503e076e5d9242fd9c5a7c843dcc470a6965d24a/src/govuk/core/_section-break.scss#L49-L51) that a GOV.UK Section Break uses – e.g.

```scss
border-bottom: 1px solid $govuk-border-colour;
```

This is the approach that was taken on the What's New page:

https://github.com/alphagov/whitehall/blob/cabec75c1951d2a5dea0d6976139d7a63f207604/app/assets/stylesheets/admin/views/_whats-new.scss#L5-L10

Reply:
Yeah, this `hr` is unfortunately a little awkward. David T brought it up as well in his review as well.
https://github.com/alphagov/whitehall/pull/7338

You're right in thinking that I tried `border-bottom` with `.app-view-edition-images__row` first. The issue is that because of the list item padding there needs to be an element within the `li` to get the border to be the correct width. This felt slightly better to me than appending some text divs at the end of the previous `li` and unfortunately using `:after` would require `calc` in the css which I don't think we quite want to get into yet.

Reply:
Very sadly, this makes sense 🙈 

Thanks for explaining it @dnkrj.